main.c

```diff
@@ -682,7 +683,12 @@ int main (int argc, char **argv) {
   sexp_scheme_init();
   res = run_main(argc, argv);
   if (sexp_fixnump(res)) {
-    return sexp_unbox_fixnum(res);
+    int code = sexp_unbox_fixnum(res);
```

Comment:
Can you ifdef this for Plan9, so users on other OSes can still specify the exit code?

Reply:
How does this option work?

That logic could probably also be abstracted away so it doesn't clutter up `main()`